integrations/quick-start-guide.md

```diff
@@ -0,0 +1,47 @@
+In this quick start guide, we will go over key features for integration management.
+
+# Integrations Page
+You can naviate to integrations page by clicking `Integrations` in the left menu bar. This page has two main sections: [add an integration](#add-an-integration "mention") and [connected integrations](#connected-integrations "mention").
+
+## Add an Integration
+To add an integration, simply click the icon for the integration you would like to connect. You would need to provide your service details together with your credentials. You could find more details [here](./adding-an-integration/) on connecting to s3, bigquery, and Google Cloud Storage (GCS).
+
+Once you added an integration, you can refer to [using integrations guide](./using-integrations/) to use the integration in Aqueduct SDK.
+
+## Connected Integrations
+You can view a list of all connected integrations. You can click into each item to view more details or manage that integration. Every Aqueduct server comes with a [demo integration](./aqueduct-demo-integration.md) that you can play around with.
+
+# Integration Detail Page
+In this page, you can view details about an integration and make changes to it:
+* [Integration Information](#integration-information "mention")
+* [Test-Connecting an Integration](#test-connecting-an-integration "mention")
+* [Editing an Integration](#editing-an-integration "mention")
+* [Deleting an Integration](#deleting-an-integration "mention")
+* [Viewing Data in an Integration](#viewing-data-in-an-integration "mention")
+* [Viewing Workflows Using an Integration](#viewing-workflows-using-an-integration "mention")
+
+## Integration Information
+At top of the integration detail page. You can view basic information about this integration. There are also a number of options to manage it, including [test-connection](#test-connecting-an-integration "mention"), [edit the integration](#editing-an-integration "mention"), and [delete the integration](#deleting-an-integration "mention").
```

Comment:
I think test, edit, and delete should be subsections of this (3 #'s instead of 2) since they are introduced here

Reply:
Once we add the 'integration management quickstart guide' title (which is level-1) we would need to down-grade all other titles. So if we want to add subsections it will become 4#'s. In this way I feel it may be better to keep the section structure flat. Also from user perspective, I feel 'actions' can be parallel to 'information' although they are technically parent-child as UI components.